AK/FloatingPointStringConversions.cpp

```diff
@@ -1527,50 +1527,127 @@ class MinimalBigInt {
+            1593091ull,
+            279109966635548161ull,
+            2554917779393558781ull,
+            14124656261812188652ull,
+            11976055582626787546ull,
+            2537941837315ull,
+            13750482914757213185ull,
+            1302999927698857842ull,
+            14936872543252795590ull,
+            2788415840139466767ull,
+            2095640732773017264ull,
+            7205570348933370714ull,
+            7348167152523113408ull,
+            9285516396840364274ull,
+            6907659600622710236ull,
+            349175ull,
+            8643096425819600897ull,
+            6743743997439985372ull,
+            14059704609098336919ull,
+            10729359125898331411ull,
+            4933048501514368705ull,
+            12258131603170554683ull,
+            2172371001088594721ull,
+            13569903330219142946ull,
+            13809142207969578845ull,
+            16716360519037769646ull,
+            9631256923806107285ull,
+            12866941232305103710ull,
+            1397931361048440292ull,
+            7619627737732970332ull,
+            12725409486282665900ull,
+            11703051443360963910ull,
+            9947078370803086083ull,
+            13966287901448440471ull,
+            121923442132ull,
+            17679772531488845825ull,
+            2216509366347768155ull,
+            1568689219195129479ull,
+            5511594616325588277ull,
+            1067709417009240089ull,
+            9070650952098657518ull,
+            11515285870634858015ull,
+            2539561553659505564ull,
+            17604889300961091799ull,
+            14511540856854204724ull,
+            12099083339557485471ull,
+            7115240299237943815ull,
+            313979240050606788ull,
+            10004784664717172195ull,
+            15570268847930131473ull,
+            10359715202835930803ull,
+            17685054012115162812ull,
+            13183273382855797757ull,
+            7743260039872919062ull,
+            9284593436392572926ull,
+            11105921222066415013ull,
+            18198799323400703846ull,
+            16314988383739458320ull,
+            4387527177871570570ull,
+            8476708682254672590ull,
+            4925096874831034057ull,
+            14075687868072027455ull,
+            112866656203221926ull,
+            9852830467773230418ull,
+            25755239915196746ull,
+            2201493076310172510ull,
+            8342165458688466438ull,
+            13954006576066379050ull,
+            15193819059903295636ull,
+            12565616718911389531ull,
+            3815854855847885129ull,
+            15696762163583540628ull,
+            805ull,
+        } };
+
+        // power_of_5[i] = 5 ** (2 ** i)
+        static constexpr auto power_of_5 = [&] {
+            constexpr size_t powers_count = 11;
+
+            Array<UnsignedStorageReadonlySpan, powers_count> result;
+            Array<size_t, powers_count> sizes = { 1, 1, 1, 1, 1, 2, 3, 5, 10, 19, 38 };
```

Comment:
Could you leave a comment or something to indicate where these sizes come from/can be calculated